engine/src/tools/debug_camera/plugin.cpp

```diff
@@ -51,7 +52,7 @@ void cubos::engine::debugCameraPlugin(Cubos& cubos)
     cubos.startupSystem("create Debug Camera").call([](Commands commands, State& debugCamera) {
         debugCamera.entity = commands.create()
                                  .named("debug-camera")
-                                 .add(Camera{.active = false})
+                                 .add(Camera{})
```

Comment:
```suggestion
                                
```
This .add(Camera{}) can be removed then just to simplify things since PerspectiveCamera already adds it